doc/04-Upgrading.md

```diff
@@ -0,0 +1,21 @@
+# Upgrading Icinga DB <a id="upgrading"></a>
+
+## Upgrading to Icinga DB RC2
+
+Icinga DB RC2 is a complete rewrite of RC1. Because of this, a lot has changed in the Redis and database schema, which
+is why they have to be deleted and recreated. The configuration file has changed from `icingadb.ini` to` icingadb.yaml`.
```

Comment:
We always use `yml` as file extension. Also, the config file itself will probably called `config.yml` unless you want to change the packaging.